src/Select/MenuList.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+  Copied as is from: https://github.com/jacobworrel/react-windowed-select/blob/master/src/MenuList.tsx
```

Comment:
I believe we need to inline the license too https://github.com/jacobworrel/react-windowed-select/blob/master/LICENSE

Reply:
Done.